tetrateci/fetch_all_tags.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+git remote add upstream https://github.com/istio/istio
+git fetch --tags origin
+git tag -l | grep -E "^[0-9]+.[0-9]+.[0-9]+$" > f1
+git fetch --tags upstream
+git tag -l | grep -E "^[0-9]+.[0-9]+.[0-9]+$" > f2
+
+echo "print f1"
+cat f1
+echo "print f2"
+cat f2
+
+tags=$(comm -13 f1 f2)
+rm f1 f2
+
+git config user.name github-actions
+git config user.email github-actions@github.com
+
+for tag in $tags; do
+    branch=$( echo $tag | rev | cut -d. -f2- | rev )
+    git checkout -b tetrate-release-$branch origin/tetrate-release-$branch
+    git merge $tag --no-edit
```

Comment:
how are we tracking the return code to check if this is successful?

Reply:
I don't think this would be necessary now after the recent changes, 😅

Reply:
We need to add return code check for git commands. Otherwise we wouldn't know if it's successful or not. For example checkout can file if there are uncommitted files or merge can fail due to conflicts. These are not thrown back to the steps to fail.

Reply:
with errexit and pipefail, the script should fail with a non zero code if any command threw a non zero code or a pipeline failed

Reply:
Gotcha, missed checking the set command